ccm_web/package.json

```diff
@@ -19,98 +19,93 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@kth/canvas-api": "^3.0.3",
+    "@kth/canvas-api": "^3.3.0-1",
```

Comment:
I guess this is still technically a pre-release. Should we wait to adopt this?

Reply:
Okay, because of the OAuth issue caused by an error type change, I had to change my approach here again. I am going with `4.0.0-0`, which required some method name changes as well.